### DISCUSSION TITLE
Welcome to pandas-profiling Discussions!

### DISCUSSION THREAD

**Discussion by gmartinsribeiro** [Data-Centric-AI-Community/fg-data-profiling#953] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AnliDuPreez** [comment]
Hi 
I am getting this error:

Summarize dataset: 95%
121/128 [38:09<52:35, 450.79s/it, Calculate cramers correlation]

C:\Users\Beastie\anaconda3\lib\site-packages\scipy\stats\stats.py:4812: RuntimeWarning: overflow encountered in longlong_scalars
  (2 * xtie * ytie) / m + x0 * y0 / (9 * m * (size - 2)))
C:\Users\Beastie\anaconda3\lib\site-packages\scipy\stats\stats.py:4814: RuntimeWarning: invalid value encountered in sqrt
  np.sqrt(var) / np.sqrt(2)))

KeyboardInterrupt

I interrupted with keyboard. How can I skip cramer's correlation so that all that progress is not lost?

Hoping to hear from you

**Reply by fabclmnt** [reply]
You can use a configuration file to deactivate the Cramer's correlation matrix computation, but setting this matrix config variable to `False`. https://pandas-profiling.ydata.ai/docs/master/pages/advanced_usage/available_settings.html

Also, this is sorted in the most recent pandas-profiling release 3.4.0! Let me know if this helped @AnliDuPreez.
